test/acceptance/client.js

```diff
@@ -246,7 +351,8 @@ describe('The node.js OC client', function(){
           components: [{
             name: 'hello-world',
             version: '~1.0.0'
-          }]
+          }],
+          parameters: {}
```

Comment:
it's because of the way `renderComponent` works (https://github.com/opentable/oc/blob/master/client/src/index.js#L26-L43). It uses `renderComponents` behind the scenes (which by default passes empty object as global parameters) - that's why `expectedRequest` contains `parameters`.

In case of the other two tests - the same thing, we're using `renderComponents` and in options (for `renderComponent`) we pass parameters which means they become global in internal `renderComponents` call.